pkgs/development/tools/misc/fujprog/default.nix

```
@@ -25,13 +26,13 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libftdi1
     libusb-compat-0_1
-  ];
+  ] ++ stdenv.lib.optionals stdenv.isDarwin darwin.apple_sdk.frameworks.IOKit;
```

```suggestion
  ] ++ stdenv.lib.optionals stdenv.isDarwin IOKit;
```

and use

```
  fujprog = callPackage ../development/tools/misc/fujprog {
    inherit (darwin.apple_sdk.frameworks) IOKit;
  };
```

in `all-packages.nix`.

---

Thank you! Fixed.